README.md

```diff
@@ -77,12 +77,12 @@ The [`Kustomize`](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kus
 
 ```yaml
 resources:
-- https://github.com/azure/aks-periscope//deployment/base?ref=<RELEASE_TAG>
```

Comment:
I _think_ the double-slash is required here: https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md#remote-directories

Does it still work if you remove it?